### PR TITLE
MNEMONIC-558: Complete the build of mnemonic-sys-vmem-service

### PR DESCRIPTION
**Pull request by yzz127**
Build for mnemonoc-sys-vmem-service

### CODE REVIEW

File: mnemonic-memory-services/mnemonic-sys-vmem-service/build.gradle [hunk 1]
```diff
@@ -15,8 +15,50 @@
...
+}
+
+shadowJar {
+  minimize()
+  destinationDirectory = file("$projectDir/../service-dist")
+  archiveClassifier = osdetector.classifier
+}
+
+task cleanDist(type: Delete) {
+    delete "$nativeDir/dist"
```
Comment (bigdata-memory) [inline mnemonic-memory-services/mnemonic-sys-vmem-service/build.gradle:54]: 2 extra white spaces here, could you please remove it? thanks.

**Review by bigdata-memory — CHANGES_REQUESTED** [review]
LGTM, just a minor issue about indentation.

File: mnemonic-memory-services/mnemonic-sys-vmem-service/build.gradle (hunk 1 above)
Comment (yzz127) [reply mnemonic-memory-services/mnemonic-sys-vmem-service/build.gradle:54]: Sure, thanks for the comments!

Comment (yzz127) [reply mnemonic-memory-services/mnemonic-sys-vmem-service/build.gradle]: Code change made, please review. Thanks!

**Review by bigdata-memory — APPROVED** [review]
LGTM, +1 thanks.

**Pull request merged**